### ISSUE TITLE
How to set Prometheus PushGateway URL

### ISSUE THREAD

**Issue by MadDataScience** [mkjoerg/nifi-prometheus-reporter#7]
METRICS_COLLECTOR_URL defaults to http://localhost:9091

How do I change that? It seems like something that should be set in nifi.properties or bootstrap.conf or some other conf file, but I can't figure out which one or where.

Thanks in advance!

**Comment by MadDataScience** [comment]
Actually, it looks like I may have gotten a bit ahead of myself. I can't even seem to get it to work locally.

I've run `docker-compose up` within `docker/prometheus/` so I've got prometheus_grafana_1, prometheus_pushgateway_1, prometheus_server_1 all running (and can see so at http://localhost:3000/?orgId=1, http://localhost:9090/ and http://localhost:9091/metrics). I ran `mvn install` to build `nifi-prometheus-nar-1.6.0.nar` and copied it into nifi-1.6.0/lib/. I ran `bin/nifi.sh restart`. I have a simple test flow that includes a GenerateFlowFile processor running. The basic metrics from the pushgateway (like go_gc_duration_seconds) seem to be there but nothing that I can see from NiFi, no FlowFilesSentLast5Minutes or BytesQueued, for instance. I think I must be missing a step, though I can't see anything in Readme.md aside from what I've already done. Any advice?

Thank you.
